site/soc/software/charts/osh/openstack-compute-kit/nova.yaml

```diff
@@ -80,15 +80,36 @@ data:
     conf:
       ceph:
         enabled: true
-        monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
+        #monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
```

Comment:
Why do we need to comment this?

Reply:
I think we need the monitors, @rajdharwadkar ?